modin/engines/base/io/text/csv_dispatcher.py

```diff
@@ -86,7 +86,11 @@ def _read(cls, filepath_or_buffer, **kwargs):
             header,
             names,
         )
-        skiprows_md, pre_reading = cls._manage_skiprows_parameter(skiprows, header_size)
+        (
+            skiprows_md,
+            pre_reading,
+            skiprows_partitioning,
```

Comment:
I'm starting to get confused between these options. Why do we need so many of them?

Reply:
Ok, i see, explanation was added to the `_manage_skiprows_parameter` docstring.